components/02-elements/image/image.hbs

```diff
@@ -1,4 +1,4 @@
 <img class="image lazyload {{ class }}"
```

Comment:
add `alt` attribute and text setting in config

Reply:
added